Sources/Parsing/Builders/OneOfBuilder.swift

```diff
@@ -18,6 +18,20 @@
 /// ```
 @resultBuilder
 public enum OneOfBuilder {
+  /// Provides support for specifying a parser in ``OneOfBuilder`` blocks.
+  @inlinable
+  public static func buildExpression<P: Parser>(_ parser: P) -> P {
+    parser
+  }
+
+  /// Provides support for specifying a substring parser in a UTF-8 ``OneOfBuilder`` block.
+  @inlinable
+  public static func buildExpression<P>(
+    _ parser: P
+  ) -> FromSubstring<Substring.UTF8View, P> {
+    FromSubstring(substringParser: parser, toSubstring: Substring.init, fromSubstring: { $0.utf8 })
```

Comment:
One (unresolved) consequence of this PR is that `FromSubstring { … }` is basically not functional in builder blocks...not sure if there's a solution to this or not, but it _effectively_ doesn't matter since you can simply interleave parsers now.